generated/nirfmxspecan/nirfmxspecan.proto

```diff
@@ -168,8 +170,6 @@ service NiRFmxSpecAn {
   rpc DPDCfgReferenceWaveform(DPDCfgReferenceWaveformRequest) returns (DPDCfgReferenceWaveformResponse);
   rpc DPDCfgReferenceWaveformSplit(DPDCfgReferenceWaveformSplitRequest) returns (DPDCfgReferenceWaveformSplitResponse);
   rpc DPDCfgSynchronizationMethod(DPDCfgSynchronizationMethodRequest) returns (DPDCfgSynchronizationMethodResponse);
-  rpc DPDCfgTargetWaveform(DPDCfgTargetWaveformRequest) returns (DPDCfgTargetWaveformResponse);
```

Comment:
Note: These renamed methods were added recently (https://github.com/ni/grpc-device/pull/1081 and https://github.com/ni/grpc-device/pull/1090) and not in the most recent release.

So no need to mark this PR as a breaking change.